doc/news/_preparation_next_release.md

```diff
@@ -48,7 +48,52 @@ docker run -it elektra/elektra
 - <<HIGHLIGHT>>
 - <<HIGHLIGHT>>
 
-### <<HIGHLIGHT>>
+### BREAKING: New header and library structure
+
+We implemented the previously decided, new [Header File Structure](/doc/decisions/6_implemented/header_file_structure.md), [Including Headers](/doc/decisions/6_implemented/header_include.md), [Library Directory Structure](/doc/decisions/6_implemented/library_directory_structure.md) and [Library Split](/doc/decisions/6_implemented/library_split.md).
+With this change it should be easier to find things in source code and know which headers are public (i.e., included in packages) and which are just internal to the repository.
+
+#### Changed headers
+
+For developers building applications/libraries that link against Elektra, this means a breaking change.
+The biggest part is the source-breaking change of the header structure.
+Please look at `src/include/elektra` to find the headers containing the symbols you need and include them in your code as e.g.
+
+```c
+#include <elektra/kdb.h>
+```
+
+Importantly, it is no longer required to set additional include paths.
+Only the standard include paths (`/usr/include`, or `/usr/local/include` when you install from source) are required.
+
+The new `src/include/elektra` folder includes all headers that Elektra installs (\*), it is copied directly to `/usr/include` on install.
+The layout of `src/include/elektra` (and consequently the layout of installed headers), follows a fixed pattern:
+
+```c
+// include all/default symbols of library libelektra-[LIB].so
+// almost all libraries provide such a header
+#include <elektra/[LIB].h>
+
+// include only foo symbols of library libelektra-[LIB].so
+// not all libraries provide modularized headers like this
+#include <elektra/[LIB]/foo.h>
+```
+
+(\*) For now, some bindings (e.g., the C++ binding) still install additional headers located in other places.
+These will also be moved to `src/include/elektra` in the future.
```

Comment:
We forgot to discuss bindings in the decision. IMO their headers should be moved to `src/include/elektra` too, so everything is in one place and we can actually install headers by simply copying a single directory. That would eliminate the risk of accidentally having stuff on the include path during build, but not after install.

However, since it wasn't discussed, I didn't move the headers for now.